CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Changelog
 
+**Unreleased**
+
+- Return global config ready status to downstream relays.  ([#2765](https://github.com/getsentry/relay/pull/2765))
+
 ## 23.11.2
 
+**Features**:
```

Comment:
Could we revert the changes in released changelogs?

Reply:
i guess not, unfixed it now!